src/aequitas/bias.py

```diff
@@ -624,14 +663,22 @@ def _get_statistical_significance(cls, original_df, disparity_df, ref_dict,
                 'get_statistical_significance: statistical significance was '
                 'not calculated. There are non-string cols within attr_cols.')
 
+        if not selected_significance:
+            binary_inclusions = {f'binary_{col}' for col in cls.significance_cols}
+        else:
+            binary_inclusions = {f'binary_{col}' for col in selected_significance}
```

Comment:
you could collapse this down to a single expression:

    binary_inclusions = {f'binary_{col}' for col in (selected_significance or cls.significance_cols)}

Reply:
My understanding is that the suggested collapse would add columns in `cls.significance_cols` even when we have `selected_significance`. We only want to check cls.significance_cols when we _don't_ have `selected_signifance`. of course, @saleiro if you disagree I'll implement that change!